tests/sklearn/test_sklearn_models.py

```diff
@@ -1230,6 +1230,9 @@ def test_serialization(
     check_serialization(model, x, use_dump_method)
 
 
+# This test is a known flaky
+# FIXME: https://github.com/zama-ai/concrete-ml-internal/issues/3932
+@pytest.mark.flaky
```

Comment:
A bit weird to flag the whole test as flaky when only one configuration of this if flaky but don't know if there is a better way 🤔